dashboard/client/src/pages/node/GPUColumn.tsx

```diff
@@ -0,0 +1,51 @@
+import { Typography } from "@material-ui/core";
+import React from "react";
+import { NodeDetail } from "../../type/node";
+import { Worker } from "../../type/worker";
+import {
+  GPU_COL_WIDTH,
+  NodeGPUEntry,
+  WorkerGPUEntry,
+} from "../dashboard/node-info/features/GPU";
+
+export const NodeGPUView = ({ node }: { node: NodeDetail }) => {
+  return (
+    <div style={{ minWidth: GPU_COL_WIDTH }}>
+      {node.gpus !== undefined && node.gpus.length !== 0 ? (
+        node.gpus.map((gpu, i) => <NodeGPUEntry gpu={gpu} slot={i} />)
```

Comment:
Could you add a `key` field to those components? Otherwise I think there may be warnings